group.hpp

```diff
@@ -77,8 +77,15 @@ class Group : public GroupInherit
     Serialize& serialize;
 
     /** @brief Custom callback when LED group is asserted
+     * Callback to lamptest requestHandler() method. Parameters and return type
```

Comment:
This is a generic callback for any LED group - please don't include lamp test in the documentation for this std::function. Instead frame the wording to be more generic.